test/e2e/db.js

```diff
@@ -57,6 +57,49 @@ const assertValidPrice = (t, p) => {
 	}
 }
 
+const assertValidTickets = (test, tickets) => {
+		test.ok(tickets);
+		for (let fare of tickets) {
+			if (fare.name !== undefined) {
+				test.equal(typeof fare.name, 'string');
+				test.ok(fare.name);
+			} else {
+				test.fail('Mandatory field "name" is missing');
+			}
+			if (fare.priceObj !== undefined) {
+				if (fare.priceObj.amount !== undefined) {
+					test.equal(typeof fare.priceObj.amount, 'number');
+					test.ok(fare.priceObj.amount > 0);
+				} else {
+					test.fail('Mandatory field "amount" in "priceObj" is missing');
+				}
+				// Check optional currency field
+				if (fare.priceObj.currency !== undefined) {
+					test.equal(typeof fare.priceObj.currency, 'string');
+				}
+			} else {
+				test.fail('Mandatory field "priceObj" in "ticket" is missing');
+			}
+			// Check optional fields
+			if (tickets.addData !== undefined) {
```

Comment:
In these cases, I prefer checking `'addData' in tickets`, as it checks if there is a field in `tickets`.

Same below.